packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -36,6 +36,10 @@ const InitialItem = ({ squad }: { squad: Squad }) => {
     origin: Origin.SquadMembersList,
   });
 
+  if (!verifyPermission(squad, SourcePermissions.Invite)) {
+    return null;
+  }
```

Comment:
should this not go above the hook, just so we don't load it unneeded?
Or is that a anti-pattern?

Reply:
Yes

Reply:
Ah yes, we don't have locked squad anymore, so please do a cleanup yes 🙏